packages/cdk/overlay/src/types.ts

```diff
@@ -0,0 +1,141 @@
+import type { Options, Placement } from '@popperjs/core'
+import type { ComponentPublicInstance, ComputedRef, Ref } from 'vue'
+
+type OverlayScrollStrategy = 'close' | 'reposition'
+type OverlayTrigger = 'click' | 'hover' | 'focus'
+type RefElement = Nullable<HTMLElement>
+type VueElement = Nullable<ComponentPublicInstance | HTMLElement>
+
+interface OverlayTriggerEvents {
+  onClick?: (event: Event) => void
+  onMouseEnter?: (event: Event) => void
+  onMouseLeave?: (event: Event) => void
+  onFocus?: (event: Event) => void
+  onBlur?: (event: Event) => void
+}
+
+interface OverlayPopperEvents {
+  onMouseEnter: () => void
+  onMouseLeave: () => void
+}
+
+interface OverlayOptions {
+  /**
+   * Control the visibility of the overlay
+   */
+  visible?: boolean
+  /* Scroll strategy for overlay */
+  scrollStrategy: OverlayScrollStrategy
+  /* Disable the overlay */
+  disable?: boolean
+  /**
+   * The distance between the arrow and the starting point at both ends.
+   *Acting when the overlay uses border-radius.
+   */
+  arrowOffset?: number
+  /* Whether to show arrow. */
+  showArrow?: boolean
+  /* Alignment of floating layer. */
+  placement: Placement
+  /**
+   * The options of popper.
+   * Used when ConnectOverlayOptions cannot meet the demand.
+   * Priority is higher than ConnectOverlayOptions.
+   */
+  popperOptions?: Partial<Options>
+  /* Trigger method. */
+  trigger: OverlayTrigger
+  /* Whether to allow the mouse to enter the overlay. */
+  allowEnter?: boolean
+  /**
+   * Overlay offset.
+   * [Horizontal axis offset, vertical axis offset]
+   */
+  offset: [number, number]
+  /**
+   * The delay of hiding overlay.
+   * Send false if you don't need it.
+   */
+  hideDelay: number | false
```

Comment:
```suggestion
  hideDelay: number 
```
0 就表示 false ？ showDelay 同理

Reply:
因为我在参考饿了么的组件设计时，他就是将0理解成我现在的false。但是我觉得不应该去掉0这个含义，因为这本身就是js的一种定义，0不应该被理解成没有延时，所以我增加了布尔值用来判断是否需要使用延时，这种做法在ant里面比较常见吧

Reply:
我建议还是去掉 `false`， 你在用的时候，判断一下 `hideDelay > 0` 不就好了。